### PR TITLE
[Naitei] ft:Login

### PR DESCRIPTION
**Pull request by ThaiKhacDuoc**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <div class="formdn__nhotoi">
+          <%= f.check_box :remember_me %>
+          <span><%= t "sessions.new.remember_me" %></span>
+        </div>
+      </div>
+      <%= f.submit t("sessions.new.login"), class: "login-button" %>
+    <p><%= t ".new_user" %><%= link_to t("sessions.new.sign_up_now"), signup_path %></p>
+    <% end %>
+  </div>
+</body>
```
Comment (TranMinhHuy2003) [inline app/views/sessions/new.html.erb]:
```suggestion
<% provide :title, t("sessions.new.title") %>
<title><%= t("sessions.new.title") %></title>
  
<body class="body-login login-page">
  <div class="formdn">
    <%= form_for :session, url: login_path do |f| %>
      <label for="" class="formdn__tieude--fontto"><%= t("sessions.new.title") %></label>
      <%= f.email_field :email, id: 'loginEmail', placeholder: t("sessions.new.email"), class: 'formdn__email' %>
      <%= f.password_field :password, id: 'loginPassword', placeholder: t("sessions.new.password"), class: 'formdn__mk' %>
      <div class="formdn__div3">
        <div class="formdn__nhotoi">
          <%= f.check_box :remember_me %>
          <span><%= t "sessions.new.remember_me" %></span>
        </div>
      </div>
      <%= f.submit t("sessions.new.login"), class: "login-button" %>
    <p><%= t ".new_user" %><%= link_to t("sessions.new.sign_up_now"), signup_path %></p>
    <% end %>
  </div>
</body>
```

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -2,28 +2,32 @@
...
+          <div class="auth-form__group">
+            <%= f.password_field :password, placeholder: t("sign_up.password"), class: "formdk__taikhoan" %>
+          </div>
+          <div class="auth-form__group">
+            <%= f.password_field :password_confirmation, placeholder: t("sign_up.password_confirmation"), class: "formdk__taikhoan" %>
+          </div>
+          <div class="auth-form__group">
+            <%= f.label :gender, t("sign_up.gender") %>
+            <%= f.select :gender, User.genders.keys.map { |gender| [t("sign_up.genders.#{gender}"), gender] }, {}, class: "formdk__gioitinh" %>
+          </div>
```
Comment (TranMinhHuy2003) [inline app/views/users/new.html.erb]:
```suggestion
        <%= form_for @user, url: signup_path do |f| %>
          <label for="" class="tieude-form"><%= t("sign_up.title") %></label>
          <%= render "shared/error_messages" %>
          <%= f.email_field :email, placeholder: t("sign_up.email"), class: "formdk__lienlac" %>
          <%= f.text_field :name, placeholder: t("sign_up.name"), class: "formdk__inputhoten" %>
          <%= f.password_field :password, placeholder: t("sign_up.password"), class: "formdk__taikhoan" %>
          <%= f.password_field :password_confirmation, placeholder: t("sign_up.password_confirmation"), class: "formdk__taikhoan" %>
          <%= f.label :gender, t("sign_up.gender") %>
          <%= f.select :gender, User.genders.keys.map { |gender| [t("sign_up.genders.#{gender}"), gender] }, {}, class: "formdk__gioitinh" %>
```

**Review by TranMinhHuy2003 — COMMENTED** [review]
cmt

**Comment by ThaiKhacDuoc** [comment]
ready

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/views/sessions/create.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#create</h1>
+<p>Find me in app/views/sessions/create.html.erb</p>
```
Comment (ghost) [inline app/views/sessions/create.html.erb]: file này không dùng thì bỏ đi em

File: config/routes.rb [hunk 4]
```diff
@@ -1,8 +1,13 @@
 Rails.application.routes.draw do
   resources :products
   scope "(:locale)", locale: /en|vi/ do
+    get 'sessions/new'
```
Comment (ghost) [inline config/routes.rb]: Dùng "

File: app/views/users/new.html.erb [hunk 5]
```diff
@@ -2,28 +2,22 @@
...
+      <div class="formdk">
+        <%= form_for @user, url: signup_path do |f| %>
+          <label for="" class="tieude-form"><%= t("sign_up.title") %></label>
+          <%= render "shared/error_messages" %>
+          <%= f.email_field :email, placeholder: t("sign_up.email"), class: "formdk__lienlac" %>
+          <%= f.text_field :name, placeholder: t("sign_up.name"), class: "formdk__inputhoten" %>
+          <%= f.password_field :password, placeholder: t("sign_up.password"), class: "formdk__taikhoan" %>
+          <%= f.password_field :password_confirmation, placeholder: t("sign_up.password_confirmation"), class: "formdk__taikhoan" %>
+          <%= f.label :gender, t("sign_up.gender") %>
+          <%= f.select :gender, User.genders.keys.map { |gender| [t("sign_up.genders.#{gender}"), gender] }, {}, class: "formdk__gioitinh" %>
```
Comment (ghost) [inline app/views/users/new.html.erb]: Em đưa vào helper nhé

File: app/views/sessions/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,20 @@
+<% provide :title, t("sessions.new.title") %>
+<title><%= t("sessions.new.title") %></title>
+
+<body class="body-login login-page">
+  <div class="formdn">
+    <%= form_for :session, url: login_path do |f| %>
+      <label for="" class="formdn__tieude--fontto"><%= t("sessions.new.title") %></label>
+      <%= f.email_field :email, id: 'loginEmail', placeholder: t("sessions.new.email"), class: 'formdn__email' %>
```
Comment (ghost) [inline app/views/sessions/new.html.erb]: Dùng "

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ThaiKhacDuoc** [comment]
ready

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 7]
```diff
@@ -1,336 +1,356 @@
...
   height: 35px;
   width: 35px;
   border-radius: 50%;
   font-size: 15px;
   cursor: pointer;
 }
-.collage-pic__preview--visible:hover, .collage-pic__heart--visible:hover, .collage-pic__bag:hover{
-  background-color: #00B207;
+
+.collage-pic__preview--visible:hover,
```
Comment (SuKay712) [inline app/assets/stylesheets/custom.scss:324]: đổi biến color

**Review by ghost — COMMENTED** [review]
merged

File: app/assets/stylesheets/custom.scss [hunk 8]
```diff
@@ -1,336 +1,356 @@
-@import 'bootstrap-sprockets';
-@import 'bootstrap';
-@import 'variable';
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "variable";
```
Comment (SuKay712) [inline app/assets/stylesheets/custom.scss:3]: dấu '

**Pull request merged**